src/js/angular/core/directives/shuttle-multiselect/shuttle-multiselect.directive.js

```diff
@@ -0,0 +1,100 @@
+import {cloneDeep} from "lodash";
```

Comment:
please add jsdocs to address the model that this directive consumes

Reply:
will add with the next MR